lisa/transformers/scripts_execution_transformer.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.operating_system import Posix
+from lisa.transformers.deployment_transformer import (
+    DeploymentTransformer,
+    DeploymentTransformerSchema,
+)
+
+
+@dataclass_json()
+@dataclass
+class ScriptEntry:
+    script: str = ""
```

Comment:
The script should be defined as an enum, we only invoke what we supported. It's for security and compatibility. For new, we have bash only. In future, maybe python, sh...

Reply:
give a default value.

Reply:
updated